pycaret/classification.py

```diff
@@ -909,8 +911,9 @@ def setup(
         raise TypeError("html parameter only accepts True or False.")
 
     # use_gpu
-    if type(use_gpu) is not bool:
-        raise TypeError("use_gpu parameter only accepts True or False.")
+    use_gpu.lower() if isinstance(use_gpu, str) else use_gpu
+    if use_gpu != "force" and type(use_gpu) is not bool:
+        raise TypeError("use_gpu parameter only accepts 'Force', True or False.")
```

Comment:
`Force` and `force` are inconsistent.

Reply:
Yeah, I suppose it would be better to keep it consistent. Fixed.